code/modules/gear_presets/uscm_ship.dm

```diff
@@ -327,7 +307,9 @@
 	)
 	assignment = JOB_CHIEF_REQUISITION
 	rank = JOB_CHIEF_REQUISITION
-	paygrade = PAY_SHORT_ME7
+	paygrades = list(PAY_SHORT_ME6 = JOB_PLAYTIME_TIER_0, PAY_SHORT_ME7 = JOB_PLAYTIME_TIER_1)
+	//unsure of this for now
+	//paygrade_high = PAY_SHORT_ME8
```

Comment:
So, anything about it? Literally SL and Warden can be same rank of QM right now, not even taking into consideration that CE got even higher ranks with this pr, and QM not even getting closer the one got stripped by Morrow.